pages/createCharacter.js

```diff
@@ -1,5 +1,5 @@
 import { Title } from '../components/header';
-import AddForm from '../components/AddForm';
+import AddForm from '../components/addForm';
 import ShowCharacter from '../components/ShowCharacter';
```

Comment:
You should decide if you want to use `camelCase` or `PascalCase` as a naming convention for your components. Don't mix them. I prefer `PascalCase` for react-components.